src/renderer/address-book/main/index.jsx

```diff
@@ -69,7 +75,7 @@ class AddressBookContainer extends Component {
 
 	render() {
 		const { classes } = this.props;
-		const { addresses } = this.state;
+		const addresses = removeDuplicates(this.state.addresses, 'address');
```

Comment:
seems very complicated, maybe better like this using lodash:
`const addresses = _.uniqueBy(this.state.addresses, 'address');`